src/ape/plugins/__init__.py

```diff
@@ -134,6 +138,10 @@ def __repr__(self):
         return f"<{self.__class__.__name__}>"
 
     def __getattr__(self, attr_name: str) -> Iterator[Tuple[str, Tuple]]:
+
+        if attr_name == "__fields_set__":
```

Comment:
I am confused as why this is needed because `PluginManager` is not a Pydantic model

Reply:
Left over from looking at the previous pydantic refactor, I'll remove it.